applications/processor/download-datablocks/src/main/java/com/csgroup/s2pdgs/downloaddatablocks/DownloadDataBlocksProcessor.java

```
@@ -211,7 +211,7 @@ private long fileSize(Path p){
                 client -> client.findById(index, sessionId, StationProduct.class)
             )
             .map(Option::of)
-            .doOnError(t -> log.error("Error reading session {} from index: {}", sessionId, index, t))
+            .doOnError(t -> log.error("Error reading session {} from index: {}", sessionId, index))
```

If the log is in error, do not swallow the exception.
Here perhaps more of a warning that an error, and so we could remove the exception

---

ok but for me , if you're looking for a id , it can be "normal" to not find it in the elastic .
so i change the log into WARN
i think refactoring the code will be nice